rero_ils/modules/notifications/jsonschemas/notifications/notification-v0.0.1.json

```diff
@@ -0,0 +1,97 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "title": "Notification",
+  "description": "JSON schema for notifications.",
+  "additionalProperties": true,
+  "required": [
+    "$schema",
+    "pid",
+    "notification_type",
+    "patron",
+    "transaction_location"
+  ],
+  "properties": {
+    "$schema": {
+      "title": "Schema",
+      "description": "Schema to validate notifications records against.",
+      "type": "string",
+      "minLength": 9,
+      "default": "https://ils.rero.ch/schema/notifications/notification-v0.0.1.json"
+    },
+    "pid": {
+      "title": "Notification ID",
+      "type": "string",
+      "minLength": 1
+    },
+    "notification_date": {
+      "type": "string",
+      "format": "date-time",
+      "title": "Notification creation date"
+    },
+    "patron": {
+      "title": "Patron",
+      "type": "object",
+      "properties": {
+        "$ref": {
+          "title": "Patron URI",
+          "type": "string",
+          "pattern": "^https://ils.rero.ch/api/patrons/.*?$"
+        }
+      }
+    },
+    "item": {
+      "title": "Item",
+      "type": "object",
+      "properties": {
+        "$ref": {
+          "title": "Item URI",
+          "type": "string",
+          "pattern": "^https://ils.rero.ch/api/items/.*?$"
+        }
+      }
+    },
+    "due_date": {
+      "type": "string",
+      "format": "date-time",
+      "title": "Checkout/renewal due date"
```

Comment:
Why not simply `"Due date"` for the title?

Reply:
good idea. I will change it.